Bottom: 5be316aefd55b80968fa5988fe99e63d9c8a20fa
Top:    83d90fae30ca5bed3cbce06c8271e1af308e5742
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-31 19:18:32 +0800

Refresh of autofs-5.0.7-teach-dumpmaps-to-output-simple-key-value-pairs.patch

---

diff --git a/lib/master.c b/lib/master.c
index bba959d..87cfc2f 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1331,16 +1331,11 @@ static void print_map_info(struct map_source *source)
 
 static int match_type(const char *source, const char *type)
 {
-	printf("match_type source %s type %s\n", source, type);
-	if (!strcmp(source, type)) {
-		printf("match\n");
+	if (!strcmp(source, type))
 		return 1;
-	}
-	/* Source file and files are synonymous */
-	if (!strncmp(source, type, 4) && (strlen(source) <= 5)) {
-		printf("match\n");
+	/* Sources file and files are synonymous */
+	if (!strncmp(source, type, 4) && (strlen(source) <= 5))
 		return 1;
-	}
 	return 0;
 }
 
@@ -1469,8 +1464,6 @@ int dump_map(struct master *master, const char *type, const char *name)
 				}
 			}
 
-			printf("instance %p\n", instance);
-
 			if (!instance) {
 				source = source->next;
 				lookup_close_lookup(ap);
